.github/workflows/e2e.yaml

```diff
@@ -20,6 +20,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Maximize build space
+      uses: ashwinath/maximize-build-space@master
```

Comment:
Oh, because what if that person decides to do something nefarious in the later commits, like exposing keys etc. Tags can also be overriden by rewriting the Git history.

Reply:
Oh in that case, you could reference the specific commit directly: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-versioned-actions

Reply:
ok i'll do that